csharp/test/Microsoft.AI.MachineLearning.Tests.CSharp/main.cs

```diff
@@ -0,0 +1,92 @@
+﻿using System;
+using System.IO;
+
+using Microsoft.AI.MachineLearning;
+using WinRT;
+
+namespace Microsoft.AI.MachineLearning.Tests
+{
+    class Program
+    {
+        static void Main(string[] args)
+        {
+            Console.WriteLine("Load squeezenet.onnx.");
+            using (var model = LearningModel.LoadFromFilePath("squeezenet.onnx"))
+            {
+               Console.WriteLine("Load kitten_224.png as StorageFile.");
+               var name = AppDomain.CurrentDomain.BaseDirectory + "kitten_224.png";
+               var image_task = Windows.Storage.StorageFile.GetFileFromPathAsync(name);
+               image_task.AsTask().Wait();
+               var image = image_task.GetResults();
+               Console.WriteLine("Load StorageFile into Stream.");
+               var stream_task = image.OpenReadAsync();
+               System.Threading.Thread.Sleep(1000);
+               // stream_task.AsTask().Wait();
+               //
+               // Unable to call AsTask on IAsyncOperation<IRandomAccessStreamWithContentType>... 
+               // System.TypeInitializationException: 'The type initializer for 'ABI.Windows.Foundation.AsyncOperationCompletedHandler`1' threw an exception.'
```

Comment:
Is there any work for us to investigate why this happens? Is this a bug in WinML?

Reply:
Its a bug in cswinrt. They are fixing it.